vignettes/dataset-status.Rmd

```diff
@@ -14,6 +14,46 @@ knitr::opts_chunk$set(
 )
 ```
 
+## Supported countries
+
+This map shows what countries have level 1 and level 2 subregion data directly from an official source within that country. Please note other countries may be provided through our interface to external data sources, such as WHO and JHU.
+
+```{r, supported_region_plot, echo=FALSE, message=FALSE}
+library(covidregionaldata)
+library(dplyr)
+library(ggplot2)
+
+regional_countries <- get_available_datasets() %>%
+  filter(type == "regional")
+
+regional_countries_l2 <- regional_countries %>%
+  filter(!(is.na(level_2_region)))
+
+world <- map_data("world")
+
+supported_countries <- mutate(
+  world,
+  fill = case_when(
+    region %in% regional_countries_l2$class ~ "Level 2",
+    region %in% regional_countries$class ~ "Level 1",
+    TRUE ~ "None"
+  )
+)
+
+ggplot(supported_countries, aes(long, lat, fill = fill, group = group)) +
+  geom_polygon(color = "black", lwd = 0.1) +
+  ggtitle("Supported countries") +
```

Comment:
I basically hate plot titles... I am happy to waive this misguided view if everyone else disagress but would probably prefer this as a legend title. On a similar note I hate legends at the top and like them at the bottom. Again personal preference so feel free to ignore :)